### DISCUSSION TITLE
Lower the feed quality for Frigate

### DISCUSSION THREAD

**Discussion by hippytyre** [mrlt8/docker-wyze-bridge#608] [Q&A]
Is there a way to lower the quality of a feed coming from wyze bridge? I'm using Frigate NVR for person detection in home assistant. I don't really need a high quality feed for this.

I've tried setting per camera using: 

- CAM_NAME: Front
  QUALITY: SD50
- CAM_NAME: Car park
  QUALITY: SD50

Are their other quality settings I can try? SD50 still seems more than I need.

Thanks very much for creating this tool, it was always annoying but being able to bring in wyze feeds.

**Comment by mrlt8** [comment]
The app can go as low as SD30, but we should be able to go lower. I'll take another look!
